CourseGrab/Supporting files/Keys.plist

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>google-client-id</key>
+	<string>894460636145-mulqctlklp1alf73u43kbdm5n7vulqga.apps.googleusercontent.com</string>
+</dict>
+</plist>
```

Comment:
same w/ this

Reply:
Oof not again... let me try to fix that...

Reply:
They should be gone now.

Reply:
if you look at the `Files Changed`, its still there. to fix it you're gonna have to:
1. Delete the files from repo, make new commit, and push commit.
2. Add files to gitignore, add files back to repo, make new commit and push commit
3. Squash all commits into one so that people can't go and look at any of your old commits where you added it.
Let me know if you need help w/ this. I can go over this w/ you in person sometime tom if you're free too. I did it w/ @ReadeP the other day

Reply:
Ok should be good now.

Reply:
It's because we moved it into a new directory, so we have to update the gitignore and add 'CourseGrab/Supporting files/Keys.plist' and 'CourseGrab/Supporting files/GoogleService-Info.plist'

Reply:
I think it should work now because I added `*Keys.plist` and `*GoogleServ...`.